paddle_fl/examples/secagg_demo/fl_master.py

```diff
@@ -0,0 +1,48 @@
+import paddle.fluid as fluid
+import paddle_fl as fl
+from paddle_fl.core.master.job_generator import JobGenerator
+from paddle_fl.core.strategy.fl_strategy_base import FLStrategyFactory
+
+class Model(object):
+    def __init__(self):
+        pass
+
+    def linear_regression(self, inputs, label):
+        param_attrs = fluid.ParamAttr(
+            name="fc_0.b_0", 
+            initializer=fluid.initializer.ConstantInitializer(0.0))
+        param_attrs = fluid.ParamAttr(
+            name="fc_0.w_0", 
+            initializer=fluid.initializer.ConstantInitializer(0.0))
+        self.predict = fluid.layers.fc(input=inputs, size=10, act='softmax', param_attr=param_attrs)
+        self.sum_cost = fluid.layers.cross_entropy(input=self.predict, label=label)
+        self.loss = fluid.layers.mean(self.sum_cost)
+        self.accuracy = fluid.layers.accuracy(input=self.predict, label=label)
+        self.startup_program = fluid.default_startup_program()
+
+inputs = fluid.layers.data(name='x', shape=[1, 28, 28], dtype='float32')
+label = fluid.layers.data(name='y', shape=[1], dtype='int64')
+
+model = Model()
+model.linear_regression(inputs, label)
+
+job_generator = JobGenerator()
+optimizer = fluid.optimizer.SGD(learning_rate=0.01)
+job_generator.set_optimizer(optimizer)
+job_generator.set_losses([model.loss])
+job_generator.set_startup_program(model.startup_program)
+job_generator.set_infer_feed_and_target_names(
+    [inputs.name, label.name], [model.loss.name])
+
+build_strategy = FLStrategyFactory()
+build_strategy.sec_agg = True
```

Comment:
you can just insert param list here into build_strategy so that sec_agg can run correctly.

Reply:
Thanks for your advise. I have inserted parameter list into strategy.